ImageDetection/ImageDetection/blob_detector.py

```diff
@@ -0,0 +1,132 @@
+import cv2
+import numpy
+import PIL.ImageOps
+from matplotlib import pyplot
+from PIL import Image
+from SyntheticDataset2 import *
+from .image_detection_settings import ImageDetectionSettings
+
+class BlobDetector(object):
+    """
+    The following code and descriptions are adapted from OpenCV:
+    """
```

Comment:
You should have a new line after the class declaration and the following code comment block.